command/server/service.go

```diff
@@ -107,3 +109,39 @@ func (s *Server) ChainSetHead(ctx context.Context, req *proto.ChainSetHeadReques
 	s.backend.APIBackend.SetHead(req.Number)
 	return &proto.ChainSetHeadResponse{}, nil
 }
+
+func ConvertBlockToBlockStub(blocks []*types.Block) []*proto.BlockStub {
```

Comment:
This needs to be exported?

Reply:
No it did not need to be exported. Fixed it !